.github/workflows/ci-windows.yml

```diff
@@ -20,10 +24,10 @@ jobs:
       - name: Install dependencies
         run: |
           nuget restore Twingly.Search.sln
-          nuget install NUnit.Console -Version 3.11.1 -OutputDirectory packages
+          cinst nunit-console-runner --version=3.11.1 --yes
```

Comment:
OK, this installation is super slow for some reason. Locally it's superfast. I'll see if something can be done, it looks like chocolatey just stalls for some reason

Reply:
Snippet from logs:

```
Fri, 18 Dec 2020 09:15:19 GMT
Chocolatey v0.10.15
Fri, 18 Dec 2020 09:17:30 GMT
Installing the following packages:
Fri, 18 Dec 2020 09:17:30 GMT
nunit-console-runner
```

Not sure what that ~2min wait time is all about.

Reply:
I am guessing it pulls a list of packages, I also noted that we have the same behavior over at Arvegods, it stalls for 1 minute or so there too.

Reply:
I will just revert to the old behavior (i.e. installing via NuGet)

Reply:
Done (via force push)